### PR TITLE
added new ui to set custom tags

### PR DESCRIPTION
**Pull request by tonymeng1**
Makes it easier to go back to videos we skipped by searching in the database for which one we want to return to. 

This commit includes ["NoCollision", "CollisionWithStationaryObject", "CollisionWithAnimal", "MultiCollision", "StationaryCamera", "ManHeldCamera", "LowQualityVideo", "LowVisibility", "PoorLighting"]

UI allows adding new tags to videos. User should commit src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache when they add a new custom tag so that everyone else will see it as well

image

### CODE REVIEW

File: src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+["NoCollision", "CollisionWithStationaryObject", "CollisionWithAnimal", "MultiCollision", "StationaryCamera", "ManHeldCamera", "LowQualityVideo", "LowVisibility", "PoorLighting"]
```
Comment (Nick-Geofroy) [inline src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache:1]: don't think this should be here right? maybe add .cache files to git ignore

**Review by Nick-Geofroy — APPROVED** [review]
minor comment but otherwise lgtm

File: src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache (hunk 1 above)
Comment (tonymeng1) [reply src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache:1]: Nah, it should be. If everyone commits their custom tags, then we won't end up with a bunch of duplicates

Comment (Nick-Geofroy) [reply src/gui_tool/gui/tinker_subuis/store_video_enum_tags.cache:1]: oh thats cool, I like that idea

**Pull request merged**